HW4.py

```diff
@@ -0,0 +1,16 @@
+# readcsv.py -path_csv example.csv -col_name Marker_Strategy
+
+import argparse
+import csv
+import sys
+
+parser = argparse.ArgumentParser()
+parser.add_argument('-path_csv')
+parser.add_argument('-col_name')
```

Comment:
Your program won't work without **path_csv** and **col_name** arguments, so they are required. So you should use positional arguments. 
If you write "-your_argument", it means that **your_argument** is optional, and your program can work without it

Also it'd be great to add some help information for users.
```
parser.add_argument('path_csv',
                    help='path to input CSV file')
parser.add_argument('col_name',
                    help='CSV column name. If column contains few words, '
                         'use _ between them')
```